src/ui/common/src/utils/storage.ts

```diff
@@ -34,3 +36,102 @@ export type StorageConfig = {
   file_config?: FileConfig;
   gcs_config?: GCSConfig;
 };
+
+export type MetadataStorageConfig = {
```

Comment:
Technically this object is really a ServerConfig.

Metadata storage config would be the StorageConfig key here.

Reply:
That makes sense--updated the naming.